cmake/BuildYARP_telemetry.cmake

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2021 iCub Facility, Istituto Italiano di Tecnologia
+# Authors: Nicolò Genesio <nicolo.genesio@iit.it>
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+include(YCMEPHelper)
+
+find_or_build_package(YCM QUIET)
+find_or_build_package(YARP QUIET)
+find_or_build_package(matioCpp QUIET)
+
+ycm_ep_helper(YARP_telemetry TYPE GIT
+              STYLE GITHUB
+              REPOSITORY robotology/yarp-telemetry.git
+              TAG master
+              COMPONENT dynamics
+              FOLDER src
+              DEPENDS YCM
+                      YARP
+                      matioCpp)
+
+set(YARP_telemetry_CONDA_DEPENDENCIES boost-cpp)
+set(YARP_telemetry_CONDA_VERSION "2.3.0.1")
```

Comment:
This is an override that should not be necessary if the correct version is added in latest releases file, did you added it for a specific reason or it is just a copy&paste leftover?

Reply:
Do you mean line 22?
Actually, it is a copy-paste and I wanted to ask you what those variables are fore 😄

Reply:
A copy-paste from where?

Reply:
Some docs are available in https://github.com/robotology/robotology-superbuild/blob/master/doc/developers-faqs.md#how-to-ensure-that-binary-packages-are-correctly-generated-for-a-new-package, but probably the docs on version are missing.

Reply:
>A copy-paste from where?

From `blocktestcore`

Reply:
Good catch, indeed the version number is wrong: https://anaconda.org/robotology/blocktest .